app/controllers/groups_controller.rb

```diff
@@ -202,14 +202,7 @@ def leave
   # DELETE /groups/1
   # DELETE /groups/1.json
   def destroy
-    # Destroy group members
-    GroupMember.where(groupid: @group.id).destroy_all
-
-    # Destroy meetings and its members
-    Meeting.where(groupid: @group.id).all.each do |meeting|
-      MeetingMember.where(meetingid: meeting.id).destroy_all
-      meeting.destroy
-    end
+    @group.destroy
```

Comment:
Who should be allowed to delete a group? Should we make sure that current_user is a leader of the group?